src/app/cdk/panel/panel/panel.component.ts

```diff
@@ -59,7 +60,10 @@ export class PanelComponent implements OnInit {
   @Input() customControls = false
   @Input() openState = true
   @Input() editable = true
+  @Input() selectable = false
+  @Input() checkbox = false
   @Output() openStateChange = new EventEmitter<boolean>()
+  @Output() checkboxChangePanel = new EventEmitter<any>()
```

Comment:
might be a good idea to define the type of the event instead of using any 

```suggestion
  @Output() checkboxChangePanel = new EventEmitter<any>()
```